# pride-resources

Since 2004, the PRoteomics IDEntifications (PRIDE) database (https://www.ebi.ac.uk/pride/) at the European Bioinformatics Institute (EMBL-EBI, Hinxton, Cambridge, UK) has enabled public data deposition of mass spectrometry (MS)-based proteomics data, providing access to the experimental data described in scientific publications. Since then, and especially in recent years, PRIDE Archive (the archival component of PRIDE) has become the largest repository for proteomics data sharing worldwide.

![m_gkab1038fig1](https://user-images.githubusercontent.com/52113/153903359-e7f83cce-51bf-4237-8cf7-84245d695e24.jpeg)


PRIDE stores datasets coming from all proteomics experimental approaches, with a focus on discovery-driven techniques such data dependent acquisition (DDA) and data independent acquisition (DIA) bottom-up proteomics, but also top-down proteomics and MS imaging, among others. For each dataset submitted to PRIDE Archive (the archival component of PRIDE), the MS raw files (output files from the mass spectrometers) and the processed results (at least peptide/protein identification results, quantification information is optional) must be provided. In addition, each dataset in PRIDE Archive can contain peptide/protein quantitation result files, the mass spectra as peak list files, the searched protein sequence databases or spectral libraries, programming scripts, and any other technical and/or biological metadata provided by the data submitters. The PRIDE team has led within the Proteomics Standards Initiative (PSI) organization, the creation and implementation of multiple standard open file formats such as mzTab, mzIdentML and mzML to store, process and visualize the proteomics data deposited.

The stand-alone ProteomeXchange (PX) Submission tool allows the researchers to perform the data submissions to PRIDE Archive, while PRIDE Inspector enables users to review the dataset before, during, and after has been deposited in the resource. After the submission is completed, different pipelines perform the validation and quality assessment of the reported results and store the data into multiple databases for enabling data access and visualization in the PRIDE Archive web interface (https://www.ebi.ac.uk/pride/archive) and also programmatically via the PRIDE Application Programming Interface (API, https://www.ebi.ac.uk/pride/ws/archive/v2/). In recent years, PRIDE Archive has been moving its visualization components from desktop-based applications (e.g., PRIDE Inspector) to Restful APIs and web-based interfaces. All submitted files are available to download via FTP or the Aspera file transfer protocol.

PRIDE resources have two main missions for the proteomics community: (i) support data deposition and quality assessment of submitted proteomics experiments, to help reproducible research; and (ii) promote and facilitate the reuse of public proteomics data, and disseminate high-quality proteomics evidences into added-value resources, including Ensembl, UniProt and Expression Atlas.

The PRIDE database was one of the founders of the PX consortium in 2011. PX defines the guidelines for data submission and dissemination of public proteomics data worldwide. As of 2021, the resources PeptideAtlas, including its related resource PASSEL (PeptideAtlas SRM Experiment Library), MassIVE, jPOST, iProX and Panorama Public are the active members of the consortium. PX coordinates the release of accession numbers for every submitted dataset and a set of services for providing unified access to publicly available datasets (http://proteomecentral.proteomexchange.org/cgi/GetDataset), including specific data types such as mass spectra, using Universal Spectrum Identifiers (http://proteomecentral.proteomexchange.org/usi/). Additionally, in 2017, PRIDE became an ELIXIR (http://www.elixir-europe.org) core data resource and ELIXIR deposition database, recognizing its key role in the life sciences.
